snuba/datasets/configuration/generic_metrics/storages/counters.yaml

```diff
@@ -71,5 +71,11 @@ allocation_policy:
       org_limit_bytes_scanned: 100000
 
 query_processors:
+  - processor: MappingOptimizer
+    args:
+      column_name: tags
+      hash_map_name: _raw_tags_hash
+      killswitch: generic_metrics/tags_hash_map_enabled
+      value_subcolumn_name: raw_value
```

Comment:
Counters doesn't have the hash map column yet so i would remove this for now. The intention is to verify this works on one dataset and then apply it to others.